src/write.fs

```diff
@@ -56,6 +56,13 @@ let getFsFileOut (fsPath: string) (tsPaths: string list) =
     let setParentNodes = true
     let host = ts.createCompilerHost(options, setParentNodes)
     let program = ts.createProgram(ResizeArray tsPaths, options, host)
+
+    // all the linked source files get loaded by TypeScript
+    // we just have to what to export and to how many files
```

Comment:
>  // we just have to what to export and to how many files


Should we add yargs option to know **how many files**?

Reply:
No, we don't need to know how many files. We need to know which files. If there is not an `--export` option then the list of `tsFiles` will be all files passed in. If one or more `--export` is set, it will be any files that match any of the "contains" filter.

Reply:
Is export option currently avaliable or should we add a PR to add export option??

Reply:
No, it needs to be added. I was hoping you could either modify the current one or create a new one.

Reply:
Ok, I will adapt the existed `multiple option` to `export option`